src/reader/ns_reader.rs

```diff
@@ -538,6 +538,10 @@ impl<'i> NsReader<&'i [u8]> {
     /// You also can use [`read_resolved_event()`] instead if you want to resolve namespace
     /// as soon as you get an event.
     ///
+    /// There is no asynchronous `read_event_async()` version of this function,
+    /// because it is meaningless to have it. You already have a buffer in a memory,
+    /// so you not needed to wait something (synchronously or asynchronously).
```

Comment:
```
    /// There is no asynchronous `read_event_async()` version of this function
    /// because it is not necessary - the contents are already in memory and no IO
    /// is necessary.
```